app/assets/javascripts/lesson_planner/unit_template_assigned.jsx

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+EC.UnitTemplatesAssigned = React.createClass({
+  propTypes: {
+    data: React.PropTypes.object.isRequired,
+    actions: React.PropTypes.object.isRequired
+  },
+
+  teacherSpecificComponents: function() {
+    var message;
+    if (this.props.actions.studentsPresent() === true) {
+      message = (
+        <span>
+          <p className='assign-success-message'>Your activity pack has been assigned.</p>
+          <div className="assign-success-button-container">
+            <a href = '/teachers/classrooms/lesson_planner'>
+              <button onClick className="button-green add-students">
+                View Assigned Activity Packs
+              </button>
+            </a>
+          </div>
+       </span>);
+    } else {
+      message = (
+        <span>
+          <p>
+            You've assigned an activity-pack to:
+              <b>{this.props.actions.getLastClassroomName()}</b>
+          </p>
+          <div className = "assignSuccess-button-container">
+            <p>Your next step is to add students.</p>
+            <a href = {this.props.actions.getInviteStudentsUrl()} >
+              <button onClick className="button-green add-students">
+                Add Students
+              </button>
+            </a>
+          </div>
+        </span>);
+    };
+    return (<div className='successBox'>
```

Comment:
@RyanNovas as a style thing - I think it's easier to read when the jsx elements (<div>, <span>, etc.) are given the shape that you would see in an html page, where the opening and closing tags are vertically aligned